### ISSUE TITLE
Block sites with host name and first path

### ISSUE THREAD

**Issue by taxen** [wildskyf/personal-blocklist#8]
This is a feature enhancement request.

There are some sites which I cannot eliminate only by host name without eliminating the whole site such as https://www.facebook.com/
I wish personal-blocklist is able to handle this situation by assigning one more information, that is the first path in the URL, such as https://www.facebook.com/abc/
In this case, all URLs from  https://www.facebook.com/ are found in the result and only URLs from https://www.facebook.com/abc/ are hidden in the result.
This must make personal-blocklist even more useful than it is.

**Comment by wildskyf** [comment]
I am thinking about changing the matching function to 'match substring', which means:

If the rule is: `google.tw/search` then:

* `google.tw` will not be hidden.
* `www.google.tw/search?q=tyui` will be hidden
* `anything.google.tw/search` will be hidden

**Comment by taxen** [comment]
Yes! That feature will satisfy my request.
Thank you in advance.

**Comment by taxen** [comment]
Updated to 2.7.0. I noticed that first page of the result has no link to eliminate. (I set 100 results per page.)
Second page and beyond have proper links to eliminate.

And how can I eliminate such result as google.tw/search with this new update?
Do I need to manually edit the block list rather than from the search result?
I hope each search result has two links, such as "block google.tw" and "block google.tw/search" in the case.

Thank you.
